pkg/util/machines/machines.go

```diff
@@ -19,7 +19,7 @@ import (
 // IsMachineHealthy returns true if the the machine is running and machine node is healthy
 func IsMachineHealthy(c client.Client, machine *mapiv1.Machine) bool {
 	if machine.Status.NodeRef == nil {
-		glog.Infof("machine %s does not have NodeRef", machine.Name)
+		glog.V(4).Infof("machine %s does not have NodeRef", machine.Name)
```

Comment:
this does not seems related to the title of this commit

Reply:
done